x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/setup_guide/setup_guide.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiSpacer, EuiTitle, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { ENTERPRISE_SEARCH_PLUGIN } from '../../../../../common/constants';
+import { SetupGuide as SetupGuideLayout } from '../../../shared/setup_guide';
+import { SetEnterpriseSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { SendEnterpriseSearchTelemetry as SendTelemetry } from '../../../shared/telemetry';
+import GettingStarted from './assets/getting_started.png';
+
+export const SetupGuide: React.FC = () => (
+  <SetupGuideLayout
+    productName={ENTERPRISE_SEARCH_PLUGIN.NAME}
+    productEuiIcon="logoEnterpriseSearch"
+    standardAuthLink="https://www.elastic.co/guide/en/app-search/current/security-and-users.html#app-search-self-managed-security-and-user-management-standard"
```

Comment:
Was it a conscious decision to direct users to App Search documentation here rather than Enterprise Search level docs?

Reply:
It's the only documentation we have for now so I am leaving it for 7.10

Reply:
Sadly we don't have shared/non-product-specific Enterprise Search auth docs yet. Chris is working on it and we should have it soon(?), and then we can hopefully get rid of the `standardAuthLink` prop altogether and keep it static :)